agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/FirstEntryPoint.java

```diff
@@ -93,6 +94,8 @@ public void init() {
 
       StartupDiagnostics.execute();
 
+      InternalLogger.initialize(Slf4jInternalLogger::create);
```

Comment:
what is this for?

Reply:
Upstream logging changed significantly in this last release: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6543